modulemd_tools/tests/test_yaml.py

```diff
@@ -0,0 +1,219 @@
+import os
+import unittest
+from unittest import mock
+import yaml
+from modulemd_tools.yaml import is_valid, validate, create, update, dump
+
+
+class TestYaml(unittest.TestCase):
+
+    def test_is_valid(self):
+        assert is_valid(yaml1)
```

Comment:
Using plain assert is not a good idea.
Asserts can be optimized out if you run python with -OO: ```python3 -OO -c "assert False; print('OK')"```
You should be using self.assert*() methods.